src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -187,6 +188,52 @@ export class MongoStorageAdapter implements StorageAdapter {
     }
     return this.client.close(false);
   }
+  async getSecurityLogs(req: any) {
+    const options = req.config || req;
+    let databaseURI = options.databaseURI;
+    const warnings = [];
+    if (databaseURI.includes('@')) {
+      databaseURI = `mongodb://${databaseURI.split('@')[1]}`;
+      const pwd = options.databaseURI.split('//')[1].split('@')[0].split(':')[1] || '';
+      if (!pwd.match('^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{14,})')) {
+        warnings.push({
```

Comment:
I think here the weakness password detection is not appropriate since developers have their own secret policies (like no special char to avoid URL issues) depending of the company. So here may be just spawn a warning if process.env.NODE_ENV === production and password not detected into the mongo uri

Reply:
In principle I agree, but Parse Server is also used by hobby developers for small projects. I think it's not a rare case that someone runs Parse Server on Heroku and connects to MongoDB Atlas, without having them isolated in a private cloud. A company with the skills to set-up a private cloud is already more security aware than a hobby developer, who is more dependent on external security guidance that his feature provides.

Maybe keep this security check and introduce an option in the future to turn off certain checks that are not required?